crates/oxc_linter/src/rules/eslint/no_unused_vars/tests/eslint.rs

```diff
@@ -364,6 +364,9 @@ fn test() {
             None,
         ),
         ("function* foo(cb) { cb = yield function(a) { cb(1 + a); }; } foo();", None), // { "ecmaVersion": 6 },
+        ("console.log(function a() {} ? b : c)", None), // { "ecmaVersion": 6 }
```

Comment:
Can you add these to oxc test cases instead? These are the tests ported directly from ESLint

Reply:
a387d57